tools/release/sync-gutenberg-packages.js

```diff
@@ -120,8 +120,8 @@ function getMismatchedNonWordPressDependencies() {
 	;
 
 	// Ensure that all the conflicts can be resolved with the same version
-	const unresolvableConflicts = Object.entries( groupBy( versionConflicts, ( [name] ) => name ) )
-		.map( ( [name, group] ) => [name, group.map( ( [, { required }] ) => required )] )
+	const unresolvableConflicts = Object.entries( groupBy( versionConflicts, ( {name} ) => name ) )
+		.map( ( [name, group] ) => [name, uniq( group.map( ( { required } ) => required ) )] )
```

Comment:
_Version_ conflicts arise when the current (_actual_) version of a package doesn't match the version that's _required_ through another dependency. If there are multiple dependencies that all require the same version, then the conflict can be resolved by upgrading (or, in rare cases, downgrading) to that version. OTOH, if there are different required versions, then the conflict is _unresolvable_.

Without the `uniq`, the code would result in false positives, such as

```
Can't resolve some conflicts automatically.
Multiple required versions of the following packages were detected:
[
  [ 'uuid', [ '8.3.2', '8.3.2', '8.3.2', '8.3.2' ] ],
  [ 'is-plain-object', [ '5.0.0', '5.0.0', '5.0.0', '5.0.0' ] ]
]
```